lib/arpv4.ml

```diff
@@ -103,6 +103,8 @@ module Make (Ethif : V1_LWT.ETHIF) (Clock : V1.CLOCK) (Time : V1_LWT.TIME) = str
     | Not_found ->
       Hashtbl.replace t.cache ip (Confirmed (expire, mac))
 
+  let add t ip mac = notify t ip mac
```

Comment:
SYNJITSU needs to do this for packets read from xenstore. I guess that's not a good idea to do it in the general case and some care should be taken even when it's read from xenstore to avoid spoofing.